velox/experimental/cudf/exec/ToCudf.cpp

```diff
@@ -491,7 +503,20 @@ CudfConfig& CudfConfig::getInstance() {
 void CudfConfig::initialize(
     std::unordered_map<std::string, std::string>&& config) {
   if (config.find(kCudfEnabled) != config.end()) {
-    enabled = folly::to<bool>(config[kCudfEnabled]);
```

Comment:
We have to change the config type to string.
I think the default semantics should be like auto unless explicitly disabled. 
So convert `enabled` -> `disabled`. disabled set to false or unset means auto.

Reply:
but of course to change _the default_, it needs to become a string.